src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -91,4 +94,52 @@ std::string HttpInfo::toJson()
   return jh.str();
 }
 
+
+
+/* ****************************************************************************
+*
+* HttpInfo::fill -
+*/
```

Comment:
Maybe `using namespace mongo` would make the fill() code cleaner.

We typically include `using namespace` in .cpp, so it wouldn't be a problem.